src/main/java/uk/gov/companieshouse/pscfiling/api/service/impl/FilingValidationServiceImpl.java

```diff
@@ -14,6 +14,9 @@
 import uk.gov.companieshouse.pscfiling.api.validator.FilingForPscTypeValid;
 import uk.gov.companieshouse.pscfiling.api.validator.FilingValidationContext;
 
+/**
+ * The filing validation service layer.
+ */
```

Comment:
ditto as above re repeating Javadoc in implementation

Reply:
Also updated.